backend/cmm_helpers.ml

```diff
@@ -891,13 +891,25 @@ let array_indexing ?typ log2size ptr ofs dbg =
    cross-compiling for 64-bit on a 32-bit host *)
 let int ~dbg i = natint_const_untagged dbg (Nativeint.of_int i)
 
-let custom_ops_unboxed_int32_odd_array =
-  Cconst_symbol
-    (Cmm.global_symbol "caml_unboxed_int32_odd_array_ops", Debuginfo.none)
+(* custom_ops_size_log2 must be kept in sync with runtime/caml/custom.h *)
```

Comment:
Can we find a programmatic way of enforcing this?

Reply:
Why do we store the log anyway ? We could store the size in words or bytes directly and do multiplications instead of shifts.

Reply:
I used a shift because I wasn't confident a div/mul would get optimized out post-cmm, but I'll check if that's actually the case.

Not sure of the best way to enforce the sizes match, though - maybe adding custom_size=64 to config.h and _Static_asserting in custom.h?

Reply:
The code-gen for `div_int` is indeed worse:
```asm
        movq	caml_unboxed_int32_array_ops@GOTPCREL(%rip), %rax
	subq	%rax, %rsi
	movq	%rsi, %rdx
	sarq	$5, %rdx
	shrq	$58, %rdx
	addq	%rdx, %rsi
	sarq	$6, %rsi
```
vs
```asm
	movq	caml_unboxed_int32_array_ops@GOTPCREL(%rip), %rsi
	subq	%rsi, %rax
	shrq	$6, %rax
```

Reply:
Why does the division matter ? I can't see any divisions added by your PR.

Reply:
Oh, sorry, I thought you were talking about emitting `lsr_int` vs `mul_int` or `div_int`. I've updated the PR to define the custom ops size in bytes instead.

Reply:
Ah no, I was indeed thinking of using multiplication and division, but failed to remember that in my second comment. I shouldn't post late in the evening... Thanks for using the size though, it makes things clearer.